packages/client/src/contents/__tests__/getCommercePages.test.js

```diff
@@ -1,18 +1,15 @@
 import { getCommercePages } from '../';
+import { mswServer } from 'tests/mswServer';
 import client from '../../helpers/client';
 import fixture from '../__fixtures__/commercepages.fixtures';
-import moxios from 'moxios';
 
 describe('getCommercePages()', () => {
   const expectedConfig = undefined;
 
   beforeEach(() => {
-    moxios.install(client);
     jest.clearAllMocks();
   });
 
-  afterEach(() => moxios.uninstall(client));
```

Comment:
I didn't tested, but I think you can call `resetHandlers()` here:
```js
afterEach(() => {
  mswServer.resetHandlers();
})
```

As mentioned in the [MSW docs](https://mswjs.io/docs/api/setup-server/reset-handlers#default-behavior):
_The .resetHandlers() function is useful as a clean up mechanism between multiple test suites that leverage runtime request handlers._